src/deployment-guides/tap.md

```diff
@@ -328,61 +280,28 @@ export TAP_REGISTRY_SERVER=<registry uri>
 export TAP_REGISTRY_USER=<registry user>
 export TAP_REGISTRY_PASSWORD=<registry password>
 export TAP_CNRS_DOMAIN=<cnrs domain>
+export alv_domain=< app live view app domain>
 
 cat <<EOF | tee tap-values-run.yaml
-profile: full
+profile: run
 ceip_policy_disclosed: true
-buildservice:
-  kp_default_repository: "${TAP_REGISTRY_SERVER}/build-service"
-  kp_default_repository_username: "${TAP_REGISTRY_USER}"
-  kp_default_repository_password: "${TAP_REGISTRY_PASSWORD}"
-  tanzunet_username: "${INSTALL_REGISTRY_USERNAME}"
-  tanzunet_password: "${INSTALL_REGISTRY_PASSWORD}"
 supply_chain: basic
-ootb_supply_chain_basic:
-  registry:
-    server: "${TAP_REGISTRY_SERVER}"
-    repository: "supply-chain"
-  gitops:
-    ssh_secret: ""
-  cluster_builder: default
-  service_account: default
-tap_gui:
-  service_type: LoadBalancer
-
-metadata_store:
-  app_service_type: LoadBalancer
-
 contour:
   infrastructure_provider: aws
   envoy:
     service:
       aws:
         LBType: nlb
 cnrs:
-  domain_name: "${TAP_CNRS_DOMAIN}"
-
-excluded_packages:
- - accelerator.apps.tanzu.vmware.com
- - api-portal.tanzu.vmware.com
- - build.appliveview.tanzu.vmware.com
- - buildservice.tanzu.vmware.com
- - controller.conventions.apps.tanzu.vmware.com
- - developer-conventions.tanzu.vmware.com
- - grype.scanning.apps.tanzu.vmware.com
- - learningcenter.tanzu.vmware.com
- - metadata-store.apps.tanzu.vmware.com
- - ootb-supply-chain-basic.tanzu.vmware.com
- - ootb-supply-chain-testing.tanzu.vmware.com
- - ootb-supply-chain-testing-scanning.tanzu.vmware.com
- - scanning.apps.tanzu.vmware.com
- - spring-boot-conventions.tanzu.vmware.com
- - tap-gui.tanzu.vmware.com
- - workshops.learningcenter.tanzu.vmware.com
+  domain_name: "${tap_run_cnrs_domain}"
```

Comment:
this should be `domain_name: "${TAP_RUN_CNRS_DOMAIN}"`

Reply:
done